include/wx/generic/private/grid.h

```diff
@@ -281,6 +281,8 @@ class WXDLLIMPEXP_ADV wxGridSubwindow : public wxWindow
 
     wxGrid *GetOwner() { return m_owner; }
 
+    virtual bool IsFrozen() const { return false; }
```

Comment:
If we do this, we should either remove or, at least, add `wxOVERRIDE`, to the existing `wxGrid{Row,Col}LabelWindow::IsFrozen()`.

Reply:
Yes, I will remove the lines from Row/Col.